### DISCUSSION TITLE
Add Diffend diffs to MR description for Ruby gems

### DISCUSSION THREAD

**Discussion by seanarnold** [renovatebot/renovate#21416] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

Currently when Rubygems get updates only the Gemfile / Gemfile.lock are changes and no further info is provided in the MR description.

Could we add a Diffend diff to the MR description (for example https://my.diffend.io/gems/devise/4.8.1/4.9.2) for Ruby gems? This would make it much easier to diff the changes.

**Comment by seanarnold** [comment]
@rarkins since both tools are Mend tools this makes sense to me.

**Reply by rarkins** [reply]
Yes, we could. You are welcome to submit a PR updating the template like is done for npm
